cmd/blzd/main.go

```diff
@@ -19,11 +19,15 @@ import (
 	"io"
 
 	"github.com/cosmos/cosmos-sdk/server"
+        "github.com/cosmos/cosmos-sdk/store"
```

Comment:
If you run go fmt before submitting, it will correct your indentations. It looks like you used space, go wants tabs.